src/OpenLoco/src/GameCommands/Track/CreateTrainStation.h

```diff
@@ -0,0 +1,42 @@
+#include "GameCommands/GameCommands.h"
+
+namespace OpenLoco::GameCommands
+{
+    struct TrackStationPlacementArgs
```

Comment:
Should we rename this, for consistency? Can be done in another PR, I'm not too fussed.
```suggestion
    struct TrainStationPlacementArgs
```

Reply:
Okay i've gone and renamed all the TrackStation things that are for train stations. There are still some trackstation things but that is the name that has been kept for things used by both road and track.